packages/server/src/migrations/functions/tableSettings.ts

```diff
@@ -0,0 +1,143 @@
+import { getScreenParams } from "../../db/utils"
```

Comment:
Unrelated to migrations but from something in the description
> If we updated table blocks to only work with plus datasource (like some other blocks do) then we could provide a binding for Clicked row ID and just use that.

Would it be possible for us to auto provide that binding only when it is a datasource plus? 
And exclude it when it's not a datasource plus?

Generally using any sort of `row._id` binding in the builder feels yucky to me when not using custom queries, potentially raising the barrier to entry for non technical folks.

Reply:
I'm not a fan either but sadly we can't - the context provided to the action handler is defined in the manifest which is static and unaware of any sort of logic like that, so it can't be conditional.

Reply:
That's sad 😔